File: front-end-web/src/app/model/PaymentRequestData.ts
import { ExpenseItem } from "./ExpenseItem";
import { Member } from "./Member";
import { PaymentRequest } from "./PaymentRequest";
import { Receipt } from "./Receipt";
import { Signature } from "./Signature";

export class PaymentRequestData {

    paymentRequest: PaymentRequest
    signatures: Array<Signature>
    items: Array<ExpenseItem>
    receipts: Array<Receipt>
}

File: web-server/src/main/java/com/churchofphilippi/webserver/model/customModels/PaymentRequestData.java
package com.churchofphilippi.webserver.model.customModels;

import com.churchofphilippi.webserver.model.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PaymentRequestData {

    private PaymentRequest paymentRequest;
    private List<ExpenseItem> items;
    private List<Receipt> receipts;
    private List<Signature> signatures;
}
